tsconfig.json

```diff
@@ -12,7 +14,7 @@
     // "declarationMap": true,                /* Generates a sourcemap for each corresponding '.d.ts' file. */
     // "sourceMap": true,                     /* Generates corresponding '.map' file. */
     // "outFile": "./",                       /* Concatenate and emit output to single file. */
-    "outDir": "./build" /* Redirect output structure to the directory. */,
+    // "outDir": "./",                        /* Redirect output structure to the directory. */
```

Comment:
Ran into circular errors in rollup saying that only one of `output.dir` or `output.file` is permitted in `rollup.config.js`, while at the same time `output.dir` is required when `"ourDir"` is specified in `tsconfig.json`. Solved by removing `"outDir"` entirely; it gets populated in `tsconfig.build.json`.